robosherlock/config/config_kinect_robot_modified.ini

```diff
@@ -0,0 +1,20 @@
+[camera]
```

Comment:
delete this file pls. We don't want to overpopulate the config folder.

Reply:
done